### PR TITLE
first contribution on develop branch

### PR DESCRIPTION
**Pull request by alghi-davide**
first contribution on develop branch

### CODE REVIEW

File: firstFixFromContributor.php [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<?php
```
Comment (davide-alghi) [inline firstFixFromContributor.php:1]: are you sure this is correct?

Comment (alghi-davide) [reply firstFixFromContributor.php:1]: yes I'm sure. tnx.

Comment (davide-alghi) [reply firstFixFromContributor.php:1]: ok resolved

**Comment by davide-alghi** [comment]
another comment

**Comment by alghi-davide** [comment]
ok read

**Comment by davide-alghi** [comment]
closed

**Pull request merged**